src/test/java/com/hpe/adm/octane/ideplugins/integrationtests/IntegrationTestSuite.java

```diff
@@ -31,15 +31,15 @@
 
 @RunWith(Suite.class)
 @Suite.SuiteClasses({
-        EntitySearchServiceITCase.class,
-        OctaneVersionServiceITCase.class,
-        CommentServiceITCase.class,
-        ConnectionSettingsITCase.class,
-        EntityLabelServiceITCase.class,
-        EntityServiceITCase.class,
-        GherkinTestDownloadITCase.class,
-        MetadataServiceITCase.class,
-        MyWorkTreeITCase.class,
+//        EntitySearchServiceITCase.class,
```

Comment:
is this normal

Reply:
i think its some github issue, mert the last commit uncomments the tests